btb/tuning/hyperparams/boolean.py

```diff
@@ -30,28 +30,30 @@ class BooleanHyperParam(BaseHyperParam):
 
     def _within_hyperparam_space(self, values):
         if values.dtype is not np.dtype('bool'):
-            raise ValueError('Value not within hyperparameter range.')
+            raise ValueError('Values: {} not within hyperparameter range.'.format(values))
 
     def _inverse_transform(self, values):
-        """Inverse trasnform one or more values from search space [0, 1]^K.
+        """Invert one or more values from search space [0, 1]^K.
 
         Converts normalized ``values`` from the search space [0, 1]^K to the original space
         of ``True`` and ``False`` by casting those values to ``boolean``.
 
         Args:
             values (ArrayLike):
-                Single value or 2D ArrayLike of normalized values.
+                2D ArrayLike of normalized values.
```

Comment:
Change the description to 2D numpy array and the type (inside the parenthesis) to `numpy.ndarray`
This should be done in all the _transform and _inverse_transform docstrings, as here only numpy arrays can be passed.